packages/bits/demo/src/components/demo/popover/popover-visual-test/popover-visual-test.component.ts

```diff
@@ -11,7 +13,10 @@ export class PopoverVisualTestComponent {
         items: ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5"],
     };
     public closePopoverSubject = new Subject();
-
+    // Testing only
```

Comment:
This is "-test.component.ts". Why the "Testing only" comment?

Reply:
Grabbed from the prev example from the comboboxv2 test